UIKitTests/Gestures/UIPanGestureRecognizerTests.swift

```diff
@@ -87,7 +87,10 @@ class UIPanGestureRecognizerTests: XCTestCase {
         // expect that state was .cancelled at some point
         wait(for: [cancelledExpectation], timeout: 1)
 
-        // state should transition to .possible after being .cancelled
+        XCTAssert(recognizer.state == .cancelled)
+
+        // state should transition to .possible after ending
+        recognizer.touchesEnded([touch], with: UIEvent())
```

Comment:
seems reasonable. does this test also run on iOS? (I remember testing touches on iOS was tricky)

Reply:
unfortunately not because the test suite is importing UIKit with `@testable`